src/store/actions/index.js

```diff
@@ -5,6 +5,7 @@ export const JOIN_HACK = "JOIN_HACK";
 export const UPDATE_HACK = "UPDATE_HACK";
 export const CLEAR_HACK = "CLEAR_HACK";
 export const FETCH_HACK_A_THON = "FETCH_HACK_A_THON";
+export const SUB_DATA = "SUB_DATA"
```

Comment:
Abbreviations like this are going to cost you in the long run when `SUB` gets overloaded to mean something completely different... eg, subscriptions to notifications about hacks.

Reply:
actions types should also (generally) be describing what you are doing with the payload, `SET_SUBMISSION_DATA`, for example.

Reply:
Ah yep, I had this thought when I wrote it, I have now changed it.